cts/cts-fencing.in

```diff
@@ -108,11 +109,35 @@ def update_path():
     os.environ['PATH'] = new_path
 
 
-def pipe_output(pipes, stdout=True, stderr=False):
+def find_validator(rng_file):
+    if os.access("/usr/bin/xmllint", os.X_OK):
+        return ["xmllint", "--relaxng", rng_file, "-"]
+    else:
+        return None
+
+
+def rng_directory():
+    if os.environ.has_key("PCMK_schema_directory"):
```

Comment:
python 3 doesn't have has_key() :)

you can use either "PCMK_schema_directory" in os.environ or try: return ... and handle the exception

Reply:
Of course it doesn't.

However, I really should be using the `in` syntax here, which appears to work for both python2 and python3.  So:
```
if "PCMK_schema_directory" in os.environ:
...
```

It's good to see that I'm forgetting all this basic python stuff after being out of it for just a few short months.